app/views/admin/api_docs/base/index.html.slim

```diff
@@ -1,27 +1,44 @@
 - content_for :page_header_title, 'ActiveDocs'
 
 - if @api_docs_services.any? { |service| service.needs_swagger_update? }
-  = link_to t("api.integrations.proxy.documentation_create_active_doc_spec", docs_base_url: docs_base_url), class: 'SwaggerUpgradeBox' do
-    p.SwaggerUpgradeBox-message
-      span.SwaggerUpgradeBox-logo
-      | ActiveDocs now supports Swagger 2.0. Read here why it makes sense to upgrade your service specs to Swagger 2.0
+  - content_for :page_header_alert do
+    br
+    div class="pf-c-alert pf-m-info pf-m-inline" aria-label="Success alert"
```

Comment:
Why "Success alert" ? (here and below also)

Reply:
Typo